setup.py

```diff
@@ -18,9 +18,9 @@ def fread(fn):
         return f.read()
 
 
-rsa_require = ['cryptography']
-signedtoken_require = ['cryptography', 'pyjwt>=1.0.0']
-signals_require = ['blinker']
+rsa_require = ['cryptography>=1.4.0']
+signedtoken_require = ['cryptography>=1.4.0', 'pyjwt>=1.6.0']
+signals_require = ['blinker>=1.4.0']
```

Comment:
Why did you change this?
Is it necessary for the feature to work?

Reply:
Seemed dangerous not to specify a version number for the _cryptography_ package, especially when there are security issues in older versions. Have only tested with the latest v2.8. Happy to remove the version numbers if that is the approach _oauthlib_ wants to take.

Reply:
Can be addressed when issue https://github.com/oauthlib/oauthlib/issues/724 is addressed.